app/views/layouts/_navbar.html.erb

```diff
@@ -4,6 +4,7 @@
     <%= link_to 'All Jobs', jobs_path, :class => 'navbar--menu__item' %>
     <%= link_to 'Add Job', new_job_path, :class => 'navbar--menu__item' %>
     <% if user_signed_in? %>
+      <%= link_to 'Add Company', new_company_path, :class => 'navbar--menu__item' %>
```

Comment:
@claeusdev While we wait to see where's the best place to put this link, can we only show it to users who haven't already registered a company? What do you think?

Reply:
@yawboakye I thought about that but i guess i had to follow the issue. 😃